### DISCUSSION TITLE
Is there any way to install Powerglitch to a shared server environment?

### DISCUSSION THREAD

**Discussion by Whimsicalcodingman** [7PH/powerglitch#21] [Q&A] [answered]
Title!

For context: I am currently working on a project on WpEngine. I do have SSH available, but unfortunately I do not have access or the possibility to install NPM packages because it is a shared server.

Is there any way to use Powerglitch by, for example, downloading all the source files and manually placing them with SFTP?

I am curious if anyone has experience with this.

👋🏼 Thanks for reading!

**Comment by 7PH** [comment, marked as answer]
Hey @Whimsicalcodingman , thanks for reaching out 

What you'll be looking for is probably the IIFE bundle, located in the repo under the dist folder. If you just copy paste this code and include it somewhere, it will register a globally available PowerGlitch object that you can directly use to glitch elements. Would that work for you?
